region/norfair/crocomire.json

```diff
@@ -1621,6 +1622,19 @@
                     "Obstacle B is present here because you need access to node 2 to bring a Mella over.",
                     "Then you need to manipulate it to go high enough to be used as a stepping stone once frozen."
                   ]
+                },
+                {
+                  "name": "Indy Jones Mella Ice Clip",
```

Comment:
Mellas take a vague number of hits to position right in my experience.  There could be ways to dodge better but I don't feel that its avoidable.